approx.jl

```diff
@@ -18,18 +18,31 @@ end
 
 rotate90(p :: Vector) = [-p[2], p[1]]
 
-function approxIpatch(P1, N1, P2, N2, points, cell)
+function approxIpatch(P1, N1, P2, N2, points, cell, modifiedBoundings = false)
```

Comment:
Why do we need the extra argument, when modified_boundings can be accessed as a global variable?

Reply:
Ah, it's in the approx package. OK.